lib/Controllers/update.js

```diff
@@ -47,10 +28,45 @@ Update.prototype.write = function(req, res, context) {
       context.attributes[a] = req.params[a];
   });
 
+  // Update associated data
+  var self = this,
+      associationPrimKeys = [];
+  if (this.include && this.include.length) {
+    this.include.forEach(function(i) {
+      var instanceModel = i.model || i,
```

Comment:
I would personally check to see if include is an instance of a Sequelize model. Also, this should have a test. [See here](https://github.com/johngiudici/epilogue/commit/c46138c55f4dd8a0b3c197f46c20b811b9851a50). Maybe this should be a separate PR if we all agree it's a good idea?